pkgs/development/coq-modules/coq-elpi/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, which, coq, ocaml, ocamlPackages }:
```

Comment:
`ocaml` is an attribute of `ocamlPackages`; `ocamlPackages` is an attribute of `coq`. You should not take them as arguments (`coq` alone is enough).